Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -76,8 +77,8 @@ bool AVIDump::Start(int w, int h)
 {
   s_pix_fmt = AV_PIX_FMT_RGBA;
 
-  s_width = w;
-  s_height = h;
+  s_width = g_Config.iDumpWidth ? g_Config.iDumpWidth : w;
```

Comment:
This part of the PR is the one I'm least comfortable with. It's working around the issue that VideoBackends don't generate the same sized frames (even when the "Crop" option is turned on, which seems to me like it should affect that). The ideal solution would be somewhere up in VideoCommon/VideoBackends.

If there wasn't the frame size problem, I also don't think Dolphin should offer an option to arbitrarily stretch output, since it's inaccurate.